packages/material-ui/src/Box/Box.test.js

```diff
@@ -19,42 +21,71 @@ describe('<Box />', () => {
     </div>
   );
 
-  it('warns if system props are used directly on the Box component', () => {
-    expect(() => {
-      render(
-        <Box
-          color="primary.main"
-          fontFamily="Comic Sans"
-          fontSize={{ xs: 'h6.fontSize', sm: 'h4.fontSize', md: 'h3.fontSize' }}
-        />,
-      );
-    }).toWarnDev('Material-UI: You are using deprecated props on the Box component.\n');
-  });
-
   it('renders children and box content', () => {
     const { container, getByTestId } = render(
-      <Box component="span" m={1}>
+      <Box component="span" sx={{ m: 1 }}>
         {testChildren}
       </Box>,
     );
     expect(container.firstChild).contain(getByTestId('child'));
     expect(container.querySelectorAll('span').length).to.equal(1);
   });
 
-  it('does not forward style props as DOM attributes', () => {
-    const elementRef = React.createRef();
-    render(
-      <Box
-        color="primary.main"
-        fontFamily="Comic Sans"
-        fontSize={{ xs: 'h6.fontSize', sm: 'h4.fontSize', md: 'h3.fontSize' }}
-        ref={elementRef}
-      />,
-    );
+  describe('warnings', () => {
+    afterEach(() => {
+      PropTypes.resetWarningCache();
+    });
+
+    it('warns if system props are used directly on the Box component', () => {
+      expect(() => {
+        PropTypes.checkPropTypes(
+          // If this breaks too often remove the test.
+          // Testing propTypes isn't worth the effort of using expando properties for internal propTypes-only stuff.
+          // eslint-disable-next-line no-underscore-dangle
+          Box.__emotion_base.propTypes,
+          {
+            color: 'primary.main',
+            fontFamily: 'Comic Sans',
+            fontSize: { xs: 'h6.fontSize', sm: 'h4.fontSize', md: 'h3.fontSize' },
+          },
+          'props',
+          'MockedName',
+        );
+      }).toErrorDev(
+        'Warning: Failed props type: The following props are deprecated: `color`, `fontFamily`, `fontSize`.',
+      );
+    });
+  });
+
+  describe('deprecated props', () => {
+    const consoleSandbox = createSandbox();
+
+    beforeEach(() => {
+      // Otherwise our global setup throws on prop-types warnings.
```

Comment:
Consider adding a comma for improved grammar:

> Otherwise, our global setup throws on prop-types warnings.

Reply:
sure

Reply:
```suggestion
      // Otherwise, our global setup throws on prop-types warnings.
```